vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -367,6 +367,18 @@
     Call<Products> getProducts(@Header("Authorization") String authHeader);
     // </editor-fold>
 
+    @PUT
+    Call<User> putContentWithUserResponse(@Header("Authorization") String authHeader,
```

Comment:
Is it weird that we have a generic URI that was return a `User`?  Should we use the `Object` versions and do a cast instead?

Reply:
If we make it return a `Call<Object>` retrofit won't deserialize the JSON response into an `User`. It will just return a `LinkedTreeMap` of the JSON content, which we will then need to manually deserialize into a `User`.

Reply:
Approving, but maybe pass this by one of the TPMs first. Since we already have the `<Object>` versions not sure if we should use them and pay the performance cost to Gson-ize from `LinkedTreeMap`->DTO in the client.

Reply:
@howardrigberg @msya do either of you think this method should be removed?

Reply:
It is a little weird as-is and clutters the API further, but until we've gotten our networking API into shape for realz I think it is a practical and simple way to solve the problem at hand.